app/src/androidTest/java/de/xikolo/testing/instrumented/unit/DeepLinkingUtilTest.kt

```diff
@@ -15,28 +15,90 @@ class DeepLinkingUtilTest : BaseTest() {
     fun testCourseIdentification() {
         assertEquals(
             "123456",
-            DeepLinkingUtil.getCourseIdentifier(Uri.parse("https://open.hpi.de/courses/123456"))
+            DeepLinkingUtil.getCourseIdentifier(
+                Uri.parse("https://open.hpi.de/courses/123456")
+            )
+        )
+
+        assertEquals(
+            "123456",
+            DeepLinkingUtil.getCourseIdentifier(
+                Uri.parse("https://open.hpi.de/courses/123456/")
+            )
+        )
+
+        assertEquals(
+            "123456",
+            DeepLinkingUtil.getCourseIdentifier(
+                Uri.parse("https://open.hpi.de/courses/123456/resume")
+            )
+        )
+
+        assertEquals(
+            "123456",
+            DeepLinkingUtil.getCourseIdentifier(
+                Uri.parse("https://open.hpi.de/courses/123456/resume/")
```

Comment:
You test the trailing `/` here?

Reply:
Yes, is there a problem with this?

Reply:
No, it just took me a minute to find the difference to the test on top. Maybe a comment about that would be helpful.

Reply:
Done